### DISCUSSION TITLE
cannot reverify student status, continue button not clickable

### DISCUSSION THREAD

**Discussion by Lucien2714** [community/community#65953] [GitHub Education]
### Hi everyone,

I verifyed my school issued email, filled up the blanks at https://education.github.com/discount_requests/application , but the continue button is still not clickable, I tried many browsers including chrome/edge/ safari on both my ipad and iphone, none of them was working. I did grant github the permisson to access my location but it was still not working. Any advice on what I should do next?

**Comment by Bonderson** [comment]
Have you found a solution to your problem?
